src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/GeneratedTestsToolWindowTest.kt

```diff
@@ -0,0 +1,149 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.uiTest
+
+import com.automation.remarks.junit5.Video
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.fixtures.EditorFixture
+import com.intellij.remoterobot.search.locators.byXpath
+import com.intellij.remoterobot.utils.WaitForConditionTimeoutException
+import com.intellij.remoterobot.utils.keyboard
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.pages.IdeaFrame
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.pages.WelcomeFrame
+import nl.tudelft.ewi.se.ciselab.testgenie.uiTest.utils.RemoteRobotExtension
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThatNoException
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.TestMethodOrder
+import org.junit.jupiter.api.extension.ExtendWith
+import java.awt.event.KeyEvent
+import java.time.Duration
+
+private const val testFileName = "ArrayUtilsTest"
+
+private const val projectName = "untitled"
+
+private const val emptyClass = """public class ArrayUtilsTest {
+}
+"""
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ExtendWith(RemoteRobotExtension::class)
+class GeneratedTestsToolWindowTest {
+
+    private lateinit var editor: EditorFixture
```

Comment:
There are some oddities on my machine (test does not pass), unfortunately

Reply:
Remind from last week: The layout of the buttons has changed from when you initially conceptualized the tests. That's why they fail, most likely.